packages/@okta/vuepress-site/code/android/index.md

```diff
@@ -68,4 +68,4 @@ Other guides:
 * [Validate access tokens](/docs/guides/validate-access-tokens)
 * [Validate ID tokens](/docs/guides/validate-id-tokens)
 
-> **Note**: Browse our [Android Developer Blog posts](/search/#q=android&f:@commonoktasource=[Developer%20blog]) for further useful topics.
+> **Note**: Browse our [Android Developer Blog posts](/blog/tags/android/) for further useful topics.
```

Comment:
```suggestion
> **Note**: Browse our [Android Developer Blog posts](https://developer.okta.com/blog/tags/android/) for further useful topics.
```

Reply:
Maybe we should use the full URL so that it opens a new tab for the blogs?

Reply:
I would suggest saying "Browse our recent [...]" since they're ordered by date descending. Also, it's possible we have posts missing tags, so let us know if you see any that might be mistagged.